climada/hazard/test/test_base.py

```diff
@@ -900,15 +904,20 @@ def test_append_raise_type_error(self):
             haz1.append(haz2)
 
     def test_concat_raise_value_error(self):
-        """Raise error if hazards with different units of type"""
-        haz1 = Hazard('TC', units='m/s')
+        """Raise error if hazards with different units, type or crs"""
+        haz1 = Hazard('TC', units='m/s',
+                      centroids=Centroids(lat=[],lon=[], crs="epsg:4326"))
         haz3 = Hazard('EQ')
         with self.assertRaises(ValueError):
             Hazard.concat([haz1, haz3])
 
         haz4 = Hazard('TC', units='cm')
         with self.assertRaises(ValueError):
             Hazard.concat([haz1, haz4])
+            
+        haz5 = Hazard('TC', centroids=Centroids(lat=[],lon=[], crs="epsg:7777"))
+        with self.assertRaises(ValueError):
```

Comment:
Please check the exact error that is raised using [`self.assertRaisesRegex`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex), where the second argument is an expression that should appear in the error message. Notice that `haz5` does not define `units`, so the error might also originate from that.